layers/cmd/export_csv.go

```diff
@@ -24,10 +24,43 @@ import (
 	lscsv "github.com/cloudprivacylabs/lsa/pkg/csv"
 )
 
+type CSVExport struct {
+	Step
+	SpecFile string
+	lscsv.Writer
+}
+
+func (ecsv CSVExport) Run(pipeline *PipelineContext) error {
+	csvExporter := ecsv.Writer
+	var spec string
+	if ecsv.SpecFile != "" {
```

Comment:
if !ecsv.initialized {
   If SpecFile not empty, read it into ecsv.Writer
  set initalized=true
}